snmp.go

```diff
@@ -92,6 +103,16 @@ func (fm *Frontman) runSNMPProbe(check *SNMPCheckData) (map[string]interface{},
 	return fm.prepareSNMPResult(check.Preset, packets)
 }
 
+// does packets contain errror oid ?
```

Comment:
I think renaming the function to something like 'findErrorPacket' or "searchForErrorInPackets" can do better documentation  than comment with a flavor of uncertainty :smile:

Reply:
True ;-)

I think the name is pretty solid "getErrorFromVariables", as it turns returned values into a golang error if a error value is found. The comment is a bit vague though.

Changed comment to
```// getErrorFromVariables returns an error if any of the oid:s in the packets contains a recognized oid error```